### ISSUE TITLE
toc-*expandable macros: move the arrow outside of the link

### ISSUE THREAD

**Issue by nameanyone** [Jermolene/TiddlyWiki5#2601]
Would it be possible to make it so that the arrow is in front of the toc link, not inside the toc link. So that when I hover over the button, the link does not get underlined.
Simply moving the `<$link>` widget opening tag works, like so:

    …
    </$reveal>
    <$link>
    <<toc-caption>>
    </$link>
    …
But that interferes with `toc-tabbed-` macros, causing the arrow to get squeezed onto a separate line.

**Comment by inmysocks** [comment]
Yes it is possible but I think that the toc macros should be updated completely sometime soon. I have the start of a new widget version as part of twederation that may work as a more flexible version that doesn't have the troubles with the transclusion. I am planning on just letting you specify a template for each item which will let you change how the toc is set up more easily.

**Comment by nameanyone** [comment]
Good to hear. If not for IE, you could make use of `<details>` tag...

http://caniuse.com/#feat=details
https://codepen.io/anon/pen/rLEmyr

**Comment by duarteframos** [comment]
Updated TOC would indeed be very useful

**Comment by pmario** [comment]
Vertical tabs are very popular. So it is very likely, that we break 3rd party stuff if we change the DOM structure too much. CSS classes shouldn't be involved, they seem to be defined outside link and image buttons. ... The only problem I have, is .. I don't see the reason. 

A weak argument would be this: 

![image](https://user-images.githubusercontent.com/374655/187100045-6637578a-08a9-41c5-9bdb-4c95dfec79e8.png)

Experimenting should be straight forward, but vertical tabs will probably need to be fixed.
